substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -197,11 +217,31 @@ impl StorageCmd {
 
 		for i in 0..self.params.warmups {
 			info!("Warmup round {}/{}", i + 1, self.params.warmups);
+			let mut child_nodes = Vec::new();
+
 			for key in keys.as_slice() {
 				let _ = client
 					.storage(hash, &key)
 					.expect("Checked above to exist")
 					.ok_or("Value unexpectedly empty");
+
+				if let Some(info) = self
+					.params
+					.include_child_trees
+					.then(|| self.is_child_key(key.clone().0))
+					.flatten()
+				{
```

Comment:
The benchmarks results are a mix/average of main and child trie accesses? The missing warm up should have slowed down the overall read performance? Does the child trie have the same amount of keys as the main trie during the benchmark?

Reply:
Not much impact because the state I run with doesn't have a significant number of child keys, it is just something I noticed while reading through the code that I think it is worth fixing.